dm/tests/validator_basic/run.sh

```diff
@@ -546,10 +546,14 @@ function stopped_validator_fail_over() {
 		"new\/ignored\/resolved: 1\/0\/0" 1 \
 		"\"stage\": \"Running\"" 2
 	# make sure validator checkpoint is flushed
+	sleep 1 # wait for the min flush interval
 	trigger_checkpoint_flush
 	run_sql_tidb_with_retry "select concat_ws('/', procd_ins, procd_upd, procd_del) processed
 														from dm_meta.test_validator_checkpoint where source='mysql-replica-01'" \
 		"processed: 3/2/1"
+	run_sql_tidb_with_retry "select count(1) cnt
```

Comment:
just noticed why there's so many spaces 😂

Reply:
copy from https://github.com/pingcap/tiflow/pull/6735.

Seems to try formatting the SQL better.

Reply:
OK, in IDE it has good indent, but on GitHub webpage it doesn't